### ISSUE TITLE
Not working for specific jpeg image

### ISSUE THREAD

**Issue by lucabezerra** [floyernick/fleep#1]
Greetings!

First of all, thanks for the great lib, it's very simple and well defined.

However, I've been having problems with a specific image, the one found in [this URL](https://assets.entrepreneur.com/content/3x2/1300/20160426154606-business-plan.jpeg). Here's the code:

```
headers = {'User-Agent': 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_11_5) ' +
               'AppleWebKit/537.36 (KHTML, like Gecko) Chrome/50.0.2661.102 Safari/537.36'}
response = requests.get(url, headers=headers)
content = ContentFile(response.content)
info = fleep.get(content.read(128))

# < ----- debug line ----- >

if not info.type_matches('raster-image'):
    raise ValidationError('some error here')
```

When debugging at the specified point of the code, I get:
```
ipdb> info.type
[]
ipdb> info.extension
[]
ipdb> info.mime
[]
```
Do you know what could be happening?

Cheers!

**Comment by ua-nick** [comment]
_Hello!_
The file has a signature that was not specified in the signature list for the jpeg format. This is fixed in version **1.0.1**.

**Comment by lucabezerra** [comment]
Oh, awesome, thanks for the quick reply!
